GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -0,0 +1,156 @@
+        * Click Add
+            * NOTE: Take note of the generated client secret (only displayed once).
+        * From the app's service blade, select Overview. Take note of the Application (client) ID for the next step.
+
+    * Grant newly created Service Principal a Contributor role to your preferred enterprise subscription
+        * Filter for subscriptions and navigate to the subscriptions list
+        * Either choose a subscription or create a new one
+        * Select your chosen subscription then select the Access control (IAM) tab from the menu blade.
+        * Click [+/Add] and select Add role assignment
+            * From the sub-menu, select 'Contributor' as a role from the drop down and search for the newly created Service Principal (i.e. cobalt-hw-admin-sp)
+            * Click Save
+            * ![image](.role-assignment-menu.png)
+
+    * Create Resource Group and Storage Account for backend state
+        * Filter for Storage accounts and navigate to the storage account list
+        * Click [+/Add] and enter values for the following fields:
+            * Subscription: Your preferred enterprise subscription for Cobalt
+            * Resource group: Create new (ex. cobalt-devint-admin-rg)
+            * Storage account name: (ex. cobalttfstates)
+        * Click [Review+Create] then [Create]
+        * Once deployment for storage account is completed, go to the resource and visit the Blobs sub-menu
+        * Click [+Container] then create a container name (ex. az-hw-remote-state-container ) with private access
+
+3. Configure Azure Devops pipeline using Azure resource values from previous steps
+
+    * Add the Azure Subscription being used for Cobalt as a *Service Connection*
+        * Return to your Azure DevOps subscription
+        * Find and select the Project Settings tab at the bottom of the screen
+        * Under the Pipelines menu select Service Connections
+        * From the Service Connections menu, select [+New Service Connection]
+        * Choose Azure Resource Manager from the dropdown and enter the Azure Subscription being used for Cobalt
+
+    * Configure *Infrastructure Pipeline Variables*
+        * Select Pipelines tab from within side-navigation menu then select Library tab
+        * Click [+Variable group] and add the following variables:
+
+            | Name   | Value | Description
+            |-------------|-----------|-----------|
+            | `AGENT_POOL` | Hosted Ubuntu 1604 | The type of build agent used for your deployment. |
+            | `ARM_PROVIDER_STRICT` | false | Terraform ARM provider modification |
+            | `BUILD_ARTIFACT_NAME` | drop | Name to identity the folder containing artifacts output by a build. |
+            | `GO_VERSION`| 1.12.5 | The version of Go terraform deployments are bound to. |
+            | `PIPELINE_ROOT_DIR` | 'devops/providers/azure-devops/templates/' | A path for finding Cobalt templates. |
```

Comment:
In the Variable Group values, the string values should not be entered with quotes. We should be more clear here or remove them.

Reply:
removed